rails_docker.rb

```diff
@@ -1,25 +1,28 @@
 require 'shellwords'
+require 'fileutils'
 
 # Add the current directory to the path Thor uses to look up files
 
 def current_directory
   @current_directory ||=
-      if __FILE__ =~ %r{\Ahttps?://}
-        require 'tmpdir'
-        tempdir = Dir.mktmpdir("rails-templates")
-        at_exit { FileUtils.remove_entry(tempdir) }
-        git clone: [
-                "--quiet",
-                "https://github.com/nimbl3/rails-templates.git",
-                tempdir
-            ].map(&:shellescape).join(" ")
-
-        tempdir
-      else
-        File.expand_path(File.dirname(__FILE__))
-      end
+    if __FILE__ =~ %r{\Ahttps?://}
+      require 'tmpdir'
+      tempdir = Dir.mktmpdir("rails-templates")
+      at_exit { FileUtils.remove_entry(tempdir) }
+      git clone: [
+              '--quiet',
+              '--depth=1',
+              'https://github.com/nimbl3/rails-templates.git',
```

Comment:
Let's fix the Github organization name to the current one:

```suggestion
              'https://github.com/nimblehq/rails-templates.git',
```